### PR TITLE
Feature/react_redux

### PR DESCRIPTION
**Pull request by kureta**
I made a React-Redux branch for your boilerplate. I had to set

```
tslint: {
        emitErrors: false,
        failOnHint: false
    }
```

because it complains about unused imports and I couldn't figure out how to fix those.

I am not exactly sure how these pull request things work but I wanted to ask you to merge this into a new `feature/react_reduce` branch. And lastly, I hope I am not bothering you with all these comments and pull requests. Have a nice day/night.

### CODE REVIEW

**Review by jwhitaker — CHANGES_REQUESTED** [review]
Take a look at https://github.com/reactjs/react-redux and examples there.  Your Counter.tsx could be simplified and broken out into separate files.

File: package.json [hunk 1]
```diff
@@ -39,6 +42,9 @@
     "webpack-dev-server": "^1.16.1"
   },
   "dependencies": {
-    "jquery": "^3.1.1"
+    "jquery": "^3.1.1",
```
Comment (jwhitaker) [inline package.json:46]: I'm wondering if jquery is even/was required at all

**Review by jwhitaker — COMMENTED** [review]
As well, we may want to add all the react/redux libraries to the vendor.bundle.js

**Comment by jwhitaker** [comment]
Not bothering me at all, I'm happy to help.

File: package.json (hunk 1 above)
Comment (kureta) [reply package.json:46]: I think not but I'm not sure.

Comment (kureta) [reply package.json:46]: just tried it. removed jquery dependency and it complained saying that "bootstrap requires jquery". I guess it was required after all.

Comment (jwhitaker) [reply package.json:46]: Of course, I should've remembered that :)

**Pull request closed without merging**